packages/headless/src/utils/action.ts

```diff
@@ -0,0 +1,11 @@
+import {AsyncThunk} from '@reduxjs/toolkit';
+
+export type AsyncThunkDefinition<
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  BaseActionType extends string,
```

Comment:
Why is this line needed if it is un-used?

Reply:
It's unused within the definition itself, but it's specified for the sole purpose of being parsed as I meant to say [here](https://github.com/coveo/ui-kit/pull/647/files#r600845682).